examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.out

```diff
@@ -1,13 +1,14 @@
 # Create a Ballerina project and a module inside it.
-# Add grpc_bidirectional_streaming.bal file inside the module directory and
-# build the project.
-# For example, if you create a module named 'service', execute the following command to build
-# the 'service' module.
-$ ballerina build service
+# Generate mock service and stub code using grpc_bidirectional_streaming.proto file.
+# For example, if you create a module named 'service', execute the following command to generate ballerina code.
+$ ballerina grpc --input grpc_bidirectional_streaming.proto --output service --mode service
+
+# Once you run the command, `grpc_bidirectional_streaming_pb.bal` and `Chat_sample_service.bal` files will be generated
```

Comment:
Let's say "... files are generated ...".  Avoid future tense wherever possible. Normally the word "will" is indicative of future tense. Apply elsewhere if applicable.

Reply:
Fixed in 71f8f2d